documentation/release_notes.markdown

```diff
@@ -118,61 +131,6 @@ Released 18 December 2018
-
-- Building Puppet Server outside our network is now slightly easier.
-
-- Prior to this release, an unnecessary and deprecated version of Facter was shipped in the `puppetserver` package. This has been removed.
-
-- Cert and CRL bundles no longer need to be in any specific order. By default, the leaf instances still come first, descending to the root, which are last. [SERVER-2465](https://tickets.puppetlabs.com/browse/SERVER-2465)
-
-## Puppet Server 6.0.4
-
-Released 26 March 2019
-
-### Bug fixes
-
-- Updated bouncy-castle to 1.60 to fix security issues. [SERVER-2431](https://tickets.puppetlabs.com/browse/SERVER-2431)
-
-## Puppet Server 6.0.3
-
-Released 15 January 2019.
-
-This release contains new features.
-
-### New Features
-
-- The `puppetserver ca` tool now respects the `server_list` setting in `puppet.conf` for those users that have created their own high availability configuration using that feature. [SERVER-2392](https://tickets.puppetlabs.com/browse/SERVER-2392) 
-- The EZBake configs now allow you to specify `JAVA_ARGS_CLI`, which is used when using `puppetserver` subcommands to configure Java differently from what is needed for the service. This was used by the CLI before, but as an environment variable only, not as an EZBake config option. [SERVER-2399](https://tickets.puppetlabs.com/browse/SERVER-2399)
-
-## Puppet Server 6.0.2
-
-Released 23 October 2018
-
-### New features
-
-- The CA service and the CA proxy service (in PE) now have their own entries in the status endpoint output and can be queried as "ca" and "ca-proxy" respectively. [SERVER-2350](https://tickets.puppetlabs.com/browse/SERVER-2350)
-
-
-## Puppet Server 6.0.1
-
-Released 2 October 2018
-
-### New features
-
-- Puppet Server now creates a default `ca.conf` file when installed, both in open source Puppet and Puppet Enterprise. CA settings such as `allow-subject-alt-names` should be configured in the `certificate-authority` section of this file. ([SERVER-2372](https://tickets.puppetlabs.com/browse/SERVER-2327))
-
-- The `puppetserver ca generate` command now has a flag `--ca-client` that will generate a certificate offline -- not using the CA API -- that is authorized to talk to that API.  This can be used to regenerate the master's host cert, or create certs for distribution to other CA nodes that need administrative access to the CA, such as the ability to sign and revoke certs. This command should only be used while Puppet Server is offline, to avoid conflicts with cert serials. ([SERVER-2320](https://tickets.puppetlabs.com/browse/SERVER-2320))
-
-- The Puppet Server CA can now sign certificates with IP alt names in addition to DNS alt names (if signing certs with alt names is enabled). ([SERVER-2267](https://tickets.puppetlabs.com/browse/SERVER-2267)
-
-
```

Comment:
Some of these notes here aren't duplicated anywhere else (at least in this file) because the Z releases went out before we had another Y stream where they could land. How do we want to handle that?

Reply:
@Magisus , going forward, we can avoid this by having all relevant fix versions on the ticket, so we can add release notes for each release stream. Obviously, we didn't have the release note process really ironed out yet, so I think we should do one of two things:

* Reconcile the missing release notes with whatever Y version they went into. I can search for which ones are missing from the file, but I would need Server team to let me know what Y versions they went into.
* Since we're trying to release this week, I could just drop this commit from the PR and we can fix it after release. Honestly, that's why I made it a separate commit. :)

Reply:
Let's drop it for now and do an audit later, as you describe in your first bullet.